addons/binding/org.openhab.binding.freebox/README.md

```diff
@@ -24,39 +79,122 @@ will be displayed in the OSGi console :
+| server        | rate_down                     | Number       | Current download  rate in byte/s                     |
+| server        | bytes_up                      | Number       | Total uploaded bytes since last connection           |
+| server        | bytes_down                    | Number       | Total downloaded  bytes since last connection        |
+| phone         | state#onhook                  | Switch       | Indicates whether the phone is on hook               |
+| phone         | state#ringing                 | Switch       | Is the phone ringing                                 |
+| phone         | any#call_number               | String       | Last call: number                                    |
+| phone         | any#call_duration             | Number       | Last call: duration in seconds                       |
+| phone         | any#call_timestamp            | DateTime     | Last call: creation timestamp                        |
+| phone         | any#call_status               | String       | Last call: type (ingoing, outgoing, missed)          |
+| phone         | any#call_name                 | String       | Last call: called name for outgoing calls. Caller name for incoming calls |
+| phone         | accepted#call_number          | String       | Last accepted call: number                           |
+| phone         | accepted#call_duration        | Number       | Last accepted call: duration in seconds              |
+| phone         | accepted#call_timestamp       | DateTime     | Last accepted call: creation timestamp               |
+| phone         | accepted#call_name            | String       | Last accepted call: caller name                      |
+| phone         | missed#call_number            | String       | Last missed call: number                             |
+| phone         | missed#call_duration          | Number       | Last missed call: duration in seconds                |
+| phone         | missed#call_timestamp         | DateTime     | Last missed call: creation timestamp                 |
+| phone         | missed#call_name              | String       | Last missed call: caller name                        |
+| phone         | outgoing#call_number          | String       | Last outgoing call: number                           |
+| phone         | outgoing#call_duration        | Number       | Last outgoing call: duration in seconds              |
+| phone         | outgoing#call_timestamp       | DateTime     | Last outgoing call: creation timestamp               |
+| phone         | outgoing#call_name            | String       | Last outgoing call: called name                      |
+| net_device    | reachable                     | Switch       | Indicates whether the network device is reachable    |
+| net_interface | reachable                     | Switch       | Indicates whether the network interface is reachable |
+
+## Example
+
+Things:
+```
```

Comment:
Could you please add an empty line here as otherwise formatting is corrupted on docs.openhab.org?